styles.css

```diff
@@ -67,3 +66,241 @@ ul {
   font-size: 20px;
   margin-top: 25px;
 }
+
+/* .........About myself section........... */
+.About-myself {
+  width: 90%;
+  margin: auto;
+}
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/206873977-1c6dd25a-1c08-4e41-92dc-b8e6593ab6c4.png)

Please make sure to use `display: flex` to style the About me section 😉.